cmd/prometheus/main.go

```diff
@@ -405,6 +407,9 @@ func main() {
 	serverOnlyFlag(a, "query.max-samples", "Maximum number of samples a single query can load into memory. Note that queries will fail if they try to load more samples than this into memory, so this also limits the number of samples a query can return.").
 		Default("50000000").IntVar(&cfg.queryMaxSamples)
 
+	serverOnlyFlag(a, "query.network", "Default network for DNS lookups. Possible values tcp (both ipv4 and ipv6), tcp4 (only ipv4), tcp6 (only ipv6).").
```

Comment:
```suggestion
	serverOnlyFlag(a, "scrape.network", "Default network for DNS lookups. Possible values tcp (both ipv4 and ipv6), tcp4 (only ipv4), tcp6 (only ipv6).").
```

Reply:
I changed this to use `scrape.network` argument and created a new field in `scrape.Options` to store this value.